README.md

```diff
@@ -49,3 +47,7 @@ dependencies:
   - numpy
   - matplotlib
 ```
+
+Only ``no-arch`` packages from ``conda-forge`` and ``emscripten-forge`` can be installed.
+- **How do I know if a package is ``no-arch`` on ``conda-forge``?** ``no-arch`` means that the package is OS-independent, usually pure-python packages are ``no-arch``. To check if your package is ``no-arch`` on ``conda-forge``, check if there is a "no-arch" entry in its feedstock repository. *e.g.* ``ipyleaflet`` is ``no-arch`` because [it specifies it here](https://github.com/conda-forge/ipyleaflet-feedstock/blob/main/recipe/meta.yaml#L14). If your package is not ``no-arch`` but is a pure Python package, then you should probably update the feedstock to turn your package into a ``no-arch`` one.
```

Comment:
Sometimes a user facing website could also be handy to find this information more easily.

For example with https://prefix.dev/channels/conda-forge/packages/ipyleaflet:

![image](https://user-images.githubusercontent.com/591645/233574457-827d4bb0-dc78-4d23-b1e3-627dc2680dcf.png)

Reply:
or with `beta.mamba.pm`:

![image](https://user-images.githubusercontent.com/591645/233574825-7fc56c55-1895-4661-b688-b2ad60db6f14.png)

Reply:
Indeed! Thanks. I will probably use your beta.mamba.pm screenshot, as I was already mentioning that website for available emscripten packages.

Reply:
Makes sense for consistency